docs/en/ingest-management/troubleshooting.asciidoc

```diff
@@ -161,6 +162,33 @@ property in the `kibana.yml` configuration file. For example:
 xpack.encryptedSavedObjects.encryptionKey: "something_at_least_32_characters"
 ----
 
+[discrete]
+[[process-not-root]]
+== {agent} fails on Windows with `Agent process is not root/admin or validation failed` message
+
+Make sure the user has administrator-level privileges.
```

Comment:
I'm not sure how generic the error is, but if it only shows up when Endpoint is used, then I think I can remove this line. WDYT?

Reply:
its not windows specific, macOS and Linux will require Administrator to run Endpoint, too